add 224-3D-segmentation (#616)
openvinotoolkit/openvino_notebooks: notebooks/224-3D-segmentation/README.md

> # 3D part segmentation

Reviewer: ```suggestion
# Part Segmentation of 3D Point Clouds with OpenVINO™ 
```

[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?labpath=notebooks%2F224-3D-segmentation%2F224-3D-segmentation.ipynb)

![3D chair](https://user-images.githubusercontent.com/91237924/185752178-3882902c-907b-4614-b0e6-ea1de08bf3ef.png)

> Point cloud is an important type of geometric data structure. Right now OpenVINO can directly consume point cloud and run inference with it.

Reviewer: ```suggestion
Point clouds are an important type of geometric data structure. OpenVINO can directly consume point cloud data and perform inference with it.
```

## Notebook Contents

> This notebook demonstrates how to process the point clould data and run 3D part segmentation with OpenVINO. The inputs of this task is a collection of individual data points in a three-dimensional plane with each point having a set coordinate on the X, Y, and Z axis.

Reviewer: ```suggestion
This notebook demonstrates how to process [point cloud](https://en.wikipedia.org/wiki/Point_cloud) data and run 3D Part Segmentation with OpenVINO. The inputs of this task are a collection of individual data points in a three-dimensional plane with each point having a set coordinates on the X, Y, and Z axes.
```
